test/integration/clusterthrottle_test.go

```diff
@@ -161,8 +162,49 @@ var _ = Describe("Clusterthrottle Test", func() {
 				Consistently(PodIsNotScheduled(ctx, DefaultNs, pod.Name)).Should(Succeed())
 			})
 		})
+		Context("Pod Group", func() {
```

Comment:
I think this is not a valid test case for the implementation.
Because the created ClusterThrottle has threshold of 2 pods, once 2 pods in `podPassedGroup` are scheduled, the ClusterThrottle reaches its threshold.
So, any pod in `podThrottledGroup` will be throttled regardless of whether kube-throttler has group-aware throttling logic.

Reply:
You have a point. I'll fix it.